readme.txt

```diff
@@ -70,7 +72,7 @@ Payment gateways supported through free or premium extensions:
 * BitPay
 * Coinbase
 
-> What can I say? Stripe is my go-to payment processor and without this add-on I don't know what I would do. Connect EDD to your Stripe account in seconds and you're done! Having a simple test mode toggle is also a valuable development tool. This is another one of my go-to EDD extensions.
+> What can I say? Stripe is my go-to payment processor and without this integration I don't know what I would do. Connect EDD to your Stripe account in seconds and you're done! Having a simple test mode toggle is also super valuable.
```

Comment:
This is a direct quote, right? Should we be using square brackets to denote our changes?

Reply:
If this is a direct quote from someone, we shouldn't modify it other than contextual hints we add, which should be in brackets.

Reply:
Assuming it's a direct quote (but I'm not sure), sure let's put "integration" in square brakets.

The second sentence can be changed back to the original.

> Having a simple test mode toggle is also a valuable development tool.

And simply leave off the 3rd sentence.

Reply:
Ok, these are direct quotes from customers. I'll remove that 3rd sentence and put `integration` in square brackets.